### DISCUSSION TITLE
Next Js 14  Facing error during build app. Failed due to Export encountered errors on following paths: in Next JS 14.

### DISCUSSION THREAD

**Discussion by girishcars24** [vercel/next.js#65401] [Help]
### Summary

Export encountered errors on following paths:
        /_not-found/page: /_not-found
        /booking/page: /booking
        /coupon/page: /coupon
        /page: /

**Comment by icyJoseph** [comment]
What errors did it output?

**Reply by yathso10** [reply]
Export encountered errors on following paths:
        /(admin-dashboard)/admin/banners/create/page: /admin/banners/create

**Comment by veerxyz** [comment]
> ### Summary
> Export encountered errors on following paths: /_not-found/page: /_not-found /booking/page: /booking /coupon/page: /coupon /page: /
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

hey were u able to rectify this?

**Comment by reinaldo-vombo** [comment]
are you using next js 14? are this page server componente. try this remove all the html from one pages like /coupon/page and leave only h2 tag then try running build and see if that its still showing error. if it fix that mean theres somthing in html that is causing that.... (sorry about ingles if write something wronge)
